ca.go

```diff
@@ -32,6 +33,7 @@ type Identity struct {
 	Province           string   `json:"province" example:"Veldhoven"`                           // Province name
 	EmailAddresses     string   `json:"email" example:"sec@company.com"`                        // Email Address
 	DNSNames           []string `json:"dns_names" example:"ca.example.com,root-ca.example.com"` // DNS Names list
+	IPAddresses        []net.IP `json:"ip_addresses,omitempty" example:"127.0.0.1,192.168.0.1"` // IP Address list
```

Comment:
!!! The comment `example:"127.0.0.1,192.168.0.1"` is imcompatible with `[]net.IP`.
The effect is, when I generate the  swaggo docs will skip the model.Identity, and print an error on the console.

Reply:
Good catch. I have put in a PR to remove the example: https://github.com/kairoaraujo/goca/pull/37